Makefile

```diff
@@ -158,11 +176,14 @@ manifests-ocp: generate-crd controller-gen kustomize
 		--set olm="${OLM}" \
 		--set autoCreateSecret=false \
 		--set createSecurityContextConstraints="true" \
-		--set operator.image="$(MASTER_IMAGE)" > config/deploy/openshift/openshift.yaml
-	grep -v 'app.kubernetes.io/managed-by' config/deploy/openshift/openshift.yaml > config/deploy/openshift/tmp.yaml
-	grep -v 'helm.sh' config/deploy/openshift/tmp.yaml > config/deploy/openshift/openshift.yaml
-	rm config/deploy/openshift/tmp.yaml
+		--set operator.image="$(MASTER_IMAGE)" > "$(OPENSHIFT_OTHERS_YAML)"
+
+	sed -i "/app.kubernetes.io\/managed-by/d" "$(OPENSHIFT_OTHERS_YAML)"
+	sed -i "/helm.sh/d" "$(OPENSHIFT_OTHERS_YAML)" 
```

Comment:
sed -i does not work on Mac

Reply:
Marking as resolved to make only single instance of this comment visible.